SpatialStructure-4/readme.md

```diff
@@ -28,13 +28,13 @@ This scenario covers the following concepts and/or IFC entities:
 
 The figure below shows the extent of the IfcSite and its georeferenced location.
 
-![](./Figure 1 - Situation.PNG)
+![](./Figure%201%20-%20Situation.PNG)
```

Comment:
Suggestion: What about not using spaces in filenames?

Reply:
Sure - will correct that.

Reply:
Fixed